src/renderers/dom/client/__tests__/ReactMount-test.js

```diff
@@ -20,9 +20,12 @@ describe('ReactMount', function() {
   var WebComponents = undefined;
 
   try {
-    WebComponents = require('WebComponents');
+    if (!/PhantomJS/.test(window.navigator.userAgent)) {
```

Comment:
No, I think the opposite is true.  We DO want to require it in any case where we're in a real browser.  The polyfill will detect if the browser supports webcomponents, and if the browser does not support them, it will polyfill them.  Either way, we want to run this test in any browser that might support webcomponents (either natively or as a result of a polyfill).

One could argue that this test is unnecessary (because webcomponents will fail to load in PhantomJS anyway), which will result in the catch block, but we do want this in real browsers.